src/events/websocket/WebSocketClients.js

```diff
@@ -249,19 +369,105 @@ export default class WebSocketClients {
       }
 
       const event = new WebSocketEvent(connectionId, route, message).create()
+      const authorizerData = this.#webSocketAuthorizersCache.get(connectionId)
+      if (authorizerData) {
+        event.identity = authorizerData.identity
+        event.authorizer = authorizerData.authorizer
+      }
       this._onWebSocketUsed(connectionId)
 
       this._processEvent(webSocketClient, connectionId, route, event)
     })
   }
 
+  _extractAuthFunctionName(endpoint) {
+    if (
+      typeof endpoint.authorizer === 'object' &&
+      endpoint.authorizer.type &&
+      endpoint.authorizer.type &&
+      endpoint.authorizer.type.toUpperCase() === 'TOKEN'
+    ) {
+      if (this.log) {
```

Comment:
Do you think we should just log here instead of erroring out? What's the reasoning behind it?

Reply:
@pgrzesik I took inspiration from HTTP implementation.
From what I can see in the other server the current authorizers behaviour is to log and ignore the authorizer if not supported.
In fact this check is only because I want to maintain a common baseline on what is not supported with the HTTP API variant using `authFunctionNameExtractor` but with Websockets I have to restrict the scope since at the moment only `REQUEST` authorizers are available.

Reply:
Thanks for clarification, it makes sense 👍